libbeat/scripts/Makefile

```diff
@@ -116,17 +116,17 @@ prepare-tests:
 # Race is not enabled for unit tests because tests run much slower.
 .PHONY: unit-tests
 unit-tests: prepare-tests
-	$(GOPATH)/bin/gotestcover -coverprofile=${COVERAGE_DIR}/unit.cov -short ${GOPACKAGES}
+	$(GOPATH)/bin/gotestcover -tags=unit $(RACE) -coverprofile=${COVERAGE_DIR}/unit.cov  ${GOPACKAGES}
```

Comment:
@urso I added the race flag to our unit-tests as I think it was actually missing. Less fun fact is that we seem to have some race conditions in libbeat: http://build-eu-00.elastic.co/job/Beats-PR/746/console